### PR TITLE
tilt: make output similar to Skaffold 

### PR DESCRIPTION
**Pull request by yuindustries**

### CODE REVIEW

File: internal/build/docker.go [hunk 1]
```diff
@@ -421,8 +422,11 @@ func readDockerOutput(ctx context.Context, reader io.Reader) (*json.RawMessage,
 			return nil, fmt.Errorf("decoding docker output: %v", err)
 		}
 
-		// TODO(Han): make me smarter! 🤓
-		logger.Get(ctx).Infof("%+v\n", message)
+		if len(message.Stream) > 0 && message.Stream != "\n" {
+			msg := strings.TrimSuffix(message.Stream, "\n")
+			logger.Get(ctx).Infof("%+v", msg)
+		}
+		// }
```
Comment (jazzdan) [inline internal/build/docker.go]: 👀

Comment (yuindustries) [reply internal/build/docker.go]: output

Comment (yuindustries) [reply internal/build/docker.go]: Does this thumbs up constitute Approval? I'm itchy to merge.

Comment (jazzdan) [reply internal/build/docker.go]: oh sorry @yuindustries, I saw that you merged branch in to master and my after 5 PM brain thought it said that you had already moved it in to master. Approving now!

**Pull request merged**